tutorial/single_pendulum/README.md

```diff
@@ -2,6 +2,8 @@
 
 ## Run model in Gazebo Garden with YARP integration
 
+For a more in-depth explanation see: https://github.com/robotology/gz-sim-yarp-plugins/issues/84
```

Comment:
This is a bit problematic, as future developers can't change or propose changes to the issue. What you can do is upload the images in an issue or a wiki, and use them in the docs, see the comment to the next change.

Reply:
Ok then I will use the issue just as a repository of images.

Reply:
Done in https://github.com/robotology/gz-sim-yarp-plugins/pull/78/commits/9e8d48dfda80e3ede76035a8d1c4dc84489139fd